### PR TITLE
Add initial version of cbdc and stablecoins slides

### PR DESCRIPTION
**Pull request by kmschuler**
# Description

Add initial version of tex slides for video "cbdc and stablecoins" including the new images used (3) and figures created (1)

## Type of Change

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/cbdc_and_stablecoins.tex [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+%%%
+\begin{frame}{References and Recommended Reading}
+		\begin{columns}[T]
+			\begin{column}{0.1\textwidth}
+					\includegraphics[width = 1.7cm, frame]{../assets/images/berentsen_schaer}
+			\end{column} %\hfill
+			\begin{column}{0.8\textwidth}
+				\textbf{The Case for Central Bank Electronic Money and the Non-case for Central Bank Cryptocurrencies} \\ 
+				Aleksander Berentsen and Fabian Schär \\
+				\link \href{https://https://research.stlouisfed.org/publications/review/2018/02/13/the-case-for-central-bank-electronic-money-and-the-non-case-for-central-bank-cryptocurrencies}{Online version}
```
Comment (dariothuerkauf) [inline slides/cbdc_and_stablecoins.tex:177]:
```suggestion
				\link \href{https://research.stlouisfed.org/publications/review/2018/02/13/the-case-for-central-bank-electronic-money-and-the-non-case-for-central-bank-cryptocurrencies}{Online version}
```

File: slides/cbdc_and_stablecoins.tex [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+%%%
+\begin{frame}{Stablecoins Overview}
+The idea of a crypto-asset representing \color{focus} a FIAT unit of currency \color{black} that has the advantages of a decentralized currency without its \color{focus}price volatility\color{black}.
+
+\vspace{1.5em}
+
+\uncover<2->{
+\textbf{Three main categories:}
+	\begin{itemize}
+		\item Off-chain collateralized, centrally issued\\
```
Comment (dariothuerkauf) [inline slides/cbdc_and_stablecoins.tex:130]:
```suggestion
		\item Off-chain collateralized, centrally issued.\\
```

**Review by dariothuerkauf — COMMENTED** [review]
Nice job! Don't know if it's a local issue, but the link to the CBDC tracker didn't work for me. Otherwise I have nothing to add 👍

**Review by Chainomics — APPROVED** [review]
Thank you @kmschuler.

**Pull request merged**